docs/data-management.rst

```diff
@@ -130,6 +130,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can organize the data in the left-hand side tree by creating new Projects, Datasets and linking the Images to those new containers.
+
+#. Use the Project |image2b| and Dataset |image2c| icons above the left hand side tree or the right-click contextual menu to create new Datasets or Projects.
+
+#. Drag and drop Images or Datasets between Datasets or Projects.
+
+#. Copy Images using the right-click context menu: 
+     - Select the images to be copied, then right-click and click ``Edit > Copy Link``
+     - Select the Dataset you want to copy the images to, right-click and click on ``Edit > Paste Link``
+
+       |image2d|
+
+.. warning::
+    **The ``Copy Link`` feature will not create a new independent copy of the images.** There is no possibility to copy images in OMERO at the moment in a way resulting in a new independent copy. All what is copied during the above action is the link between the images and the dataset (i.e. now the same image(s) is(are) linked to two datasets). 
+    
+    If you delete one of the datasets, the second dataset will retain the images. Nevertheless, **if you directly select and delete the images with two links themselves, the images will be deleted.** There is a clear warning in the OMERO.web when you try to delete such doubly-linked image, see screenshot below.
+
```

Comment:
If you delete one of the Datasets, any Images within it that are linked to other Datasets will be retained. Nevertheless, **if you directly select and delete an Image that has been copied from another Dataset it will be deleted and lost from both Datasets**.

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/75e6ba227daab5caae530f346e9b5964bbc229f0